dolphin/processor/config.py

```diff
@@ -386,6 +385,44 @@ def get_masks(self):
 
                         for extra_region in extra_masked_regions:
                             mask *= extra_region
+                        # Mask Edge Pixels
+                        try:
+                            self.settings['mask']['mask_edge_pixel']
+                        except (NameError, KeyError):
+                            pass
+                        else:
+                            bdr_len = \
```

Comment:
Please use full words for variable names whenever possible, it makes the code much more readable. I assume, this variable name should be `border_length`.